src/components/atoms/LanguageOption/index.tsx

```diff
@@ -1,16 +1,34 @@
-import * as React from 'react'
-import { StyleLanguageOptionContainer, StyleLanguageOption } from "./styles"
+import React, {useState} from 'react'
+import { StyleLanguageOptionContainer, StyleLanguageOption, StyleLanguageLink } from "./styles"
+import { americanFlagSvg, japanFlagSvg } from "../../../Svgs"
+import { useGetRedirectLanguage } from "../../../../localeRedirect"
 
 interface LanguageOptionInterface {
     language: string
 }
 
+const setLangVer = (props) => {
+    if (props.forcedLanguage === undefined || "ja") {
+        props.forcedLanguage = "en"
+        props.language = "en";
+        console.log("CHANGED PROPS: ", props.forcedLanguage)
+    } 
+    if (props.forcedLanguage === "en") {
+        props.forcedLanguage = "ja"
+        props.language = "ja";
+        console.log("CHANGED PROPS: ", props.forcedLanguage)
+    } 
+
+}
+
 const LanguageOption: React.FC<LanguageOptionInterface> = (
     props: LanguageOptionInterface
 ) => {
     return (
         <StyleLanguageOptionContainer>
             <StyleLanguageOption>{props.language}</StyleLanguageOption>
+            {props.language === "en" && <StyleLanguageLink to={"/ja"} onClick={setLangVer}>{japanFlagSvg}</StyleLanguageLink>}
```

Comment:
Instead of 
```
  {props.language === "en" && <StyleLanguageLink to={"/ja"} onClick={setLangVer}>{japanFlagSvg}</StyleLanguageLink>}
            {props.language === "ja" && <StyleLanguageLink to={"/"} onClick={setLangVer}>{americanFlagSvg}</StyleLanguageLink>}

```

You can do : 

```
  {props.language === "en" ? <StyleLanguageLink to={"/ja"} onClick={setLangVer}>{japanFlagSvg}</StyleLanguageLink> :   <StyleLanguageLink to={"/"} onClick={setLangVer}>{americanFlagSvg}</StyleLanguageLink>}

```

Reply:
Also, there is a potential bug here. 

`to` should come from props. 

What's if there user is on `/about` and clicks on the language switcher? It needs to be redirected to `/ja/about` not to `/ja`

Reply:
Took me ages, but I think I figured it out lol